apps/payment/models.py

```diff
@@ -298,9 +297,32 @@ def save(self, *args, **kwargs):
                 raise NotAcceptable("Insufficient funds")
 
             self.user.save()
-        super(PaymentTransaction, self).save(*args, **kwargs)
+        super().save(*args, **kwargs)
 
     class Meta:
         ordering = ['-datetime']
         verbose_name = _('transaksjon')
         verbose_name_plural = _('transaksjoner')
+
+
+class PaymentReceipt(models.Model):
+    """Transaction receipt"""
+    receipt_id = models.CharField(max_length=128, default=None)
+    to_mail = models.EmailField()
+    from_mail = models.EmailField()
+    subject = models.TextField()
+    description = models.TextField()
+    transaction_date = models.DateTimeField()
```

Comment:
If you connect the receipt to a specific payment you could get a lot of these fields for free. Event payments have user (from which you can get to_email), subject (event title), description (what you purchased) [or the other way around], as well as the transaction date.

Reply:
(And when I come to think of it, calling save() on a payment could actually automagically create a PaymentReceipt, which is automagically sent to the user. Neat, huh?)